runtime/cere-dev/src/lib.rs

```diff
@@ -149,7 +149,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// and set impl_version to 0. If only runtime
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
-	spec_version: 54112,
+	spec_version: 54113,
 	impl_version: 0,
 	apis: RUNTIME_API_VERSIONS,
 	transaction_version: 19,
```

Comment:
You have changed existing extrinsics. The `transaction_version` should be increased..

Reply:
extrensics were not changed as part of this release and will change in the next one